modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -0,0 +1,120 @@
+<template>
```

Comment:
I don't know if this is related to this new component, but you type in the "slug" field, the "title" field becomes `undefined` (the value in the Vue component, not the rendered input in the DOM) and vice versa.

Reply:
You saw that using this branch? I'm not sure about that. This is simply a placeholder component to avoid all of the errors since it was missing. This component should not be considered complete at all.